Answers.md

```diff
@@ -1,3 +1,29 @@
-### Your responses to the short answer questions should be laid out here using markdown.
+# Describe the differences between a Docker container and a virtual machine. What makes a container more aptly-suited for portability?
 
-For help with markdown syntax, [go here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
+A Docker container is an image that contains everything needed to run some software whereas a virtual machine is an entire os running on another os. Both Docker and virtual machines isolate themselves from the OS they are running on, but docker itself is just software being executed by the OS.
```

Comment:
A virtual machine in the end is also just software running being executed by an OS. Docker is more portable also due to the fact that a lot of the nested operating system commands that a virtual machine executes are 'outsourced' by the Docker engine that runs locally on each machine. This is a big part of why Docker containers are so much more lightweight in comparison to VMs.